### DISCUSSION TITLE
Error: Error parsing info: Unable to retrieve video metadata

### DISCUSSION THREAD

**Discussion by mihawkman** [eritislami/evobot#387] [Q&A]
The bot nonstop giving me `Error: Error parsing info: Unable to retrieve video metadata` why?

**Comment by aka-pal** [comment]
Restart it

**Reply by mihawkman** [reply]
i did and still doesnt work

**Reply by totolr** [reply]
I have the same problem as you.

**Reply by totolr** [reply]
did you find a solution ?

**Reply by mihawkman** [reply]
nope

**Reply by Undermaks** [reply]
I have same problem
